test/test.js

```diff
@@ -208,6 +208,15 @@ describe('parser (exif data)', () => {
 			'003b.jpg',
 			'exif-js-issue-124.tiff',
 			'noexif.jpg',
+			'DJI_Zenmuse_XT2_640_R_13mm.jpg',
+			'DJI_Zenmuse_XT2_640_R_13mm.tiff',
+			'FLIR_DUO_PRO_640_R_13mm.JPG',
+			'FLIR_DUO_PRO_640_R_13mm.TIFF',
+			'MicaSense_RedEdge_M.tif',
+			'Delair_UX11.jpg',
+			'FLIR_Tau_2_640_R_19mm.jpg',
+			'Sensefly_Thermomap_9mm.tif',
+            'Wiris_2nd_Gen_640_19mm.tiff',
```

Comment:
There are some more cameras in our sample data, but this already covers the most important ones.